addons/web_tour/static/tests/tour_manager_tests.js

```diff
@@ -147,5 +147,46 @@ odoo.define('web_tour.tour_manager_tests', async function (require) {
 
             tourManager.destroy();
         });
+
+        QUnit.test("Anchor repositioning", async function (assert) {
+            assert.expect(3);
+
+            const tourManager = await createTourManager({
+                observe: true,
+                template: '<div class="wrapper" style="padding: 50px 0px;"><input class="anchor"/></div>',
+                tours: [{
+                    name: "Tour",
+                    options: { rainbowMan: false },
+                    steps: [{
+                        trigger: "input.anchor",
+                        position: "top"
+                    }],
+                }],
+                debug: true,
```

Comment:
debug ?

Reply:
This is one of those "magic boolean" that you enable because it doesn't work without it.

More seriously, I copied other tests for that and when digging a bit, it seems related to the fact that is doesn't actually put the tip in the DOM during the test if you don't set that boolean to True? Which in turn doesn't allow me to correctly test the positioning.

I added a comment similarly to other tests to make it a bit more clear.